src/LinAlg/hiopMatrixRajaSparseTriplet.hpp

```diff
@@ -155,14 +155,16 @@ class hiopMatrixRajaSparseTriplet : public hiopMatrixSparse
 
   virtual void copySubmatrixFrom(const hiopMatrix& src_gen,
                                    const long long& dest_row_st, const long long& dest_col_st,
-                                   const long long& dest_nnz_st)
+                                   const long long& dest_nnz_st,
+                                   const bool offdiag_only = false)
   {
     assert(false && "not needed / implemented");
   }
     
   virtual void copySubmatrixFromTrans(const hiopMatrix& src_gen,
                                    const long long& dest_row_st, const long long& dest_col_st,
```

Comment:
should we split this line to two lines?

Reply:
oops I forgot to fix this... will fix it in a branch I am working now and update it later since I don't want to push too many times to `develop`... this PR was not supposed to be updated :)